crud/replace.lua

```diff
@@ -66,9 +66,8 @@ local function call_replace_on_router(vshard_router, space_name, original_tuple,
 
     local space, err = utils.get_space(space_name, vshard_router:routeall())
     if err ~= nil then
-        return nil, ReplaceError:new("Failed to get space %q: %s", space_name, err), const.NEED_SCHEMA_RELOAD
+        return nil, err, const.NEED_SCHEMA_RELOAD
```

Comment:
I think all errors returned from `replace` should be of the `ReplaceError` class, at least it works like this for any other errors and we don't planned to rework it anytime soon.

Same for other API methods.

Reply:
I propose to consider this:

```lua
return nil, InsertError:new("An error occurred during the operation: %q", err.str), const.NEED_SCHEMA_RELOAD
```

Output from this:

```yaml
myapp.router> crud.insert('customer',{2,100,'Mike'})
---
- null
- line: 69
  class_name: InsertError
  err: 'An error occurred during the operation: "GetSpaceError: The connection to
    the master of replicaset 548fb6ff-d686-4298-ad17-7f81fa19588c is not valid: connect,
    called on fd 37, aka 127.0.0.1:51219: Connection refused"'
  file: '...r_bug/repro/myapp/.rocks/share/tarantool/crud/insert.lua'
  str: 'InsertError: An error occurred during the operation: "GetSpaceError: The connection
    to the master of replicaset 548fb6ff-d686-4298-ad17-7f81fa19588c is not valid:
    connect, called on fd 37, aka 127.0.0.1:51219: Connection refused"'
```

Reply:
Regarding `%q` and `err.str` -- I think for now you should simply stick to the existing wrappers.

Reply:
Done, now using `%s` and `err`